src/DynamoUtilities/CertificateVerification.cs

```diff
@@ -37,7 +38,17 @@ public static bool CheckAssemblyForValidCertificate(string assemblyPath)
             }
 
             //Verify the node library has a verified signed certificate
-            var cert = asm.Modules.FirstOrDefault()?.GetSignerCertificate();
+            X509Certificate cert;
```

Comment:
I think it may be even safer to check this before loading the asm on line 32. (Just a comment)

Reply:
FYI: @saintentropy What do you think? Would you consider such change in `master` branch?